src/main/java/jfxtras/labs/internal/scene/control/skin/LedBargraphSkin.java

```diff
@@ -139,12 +139,18 @@ public void changed(ObservableValue<? extends Number> ov, Number oldValue, Numbe
                 int currentLedPeakIndex = 0;
                 for (int i = 0 ; i < control.getNoOfLeds() ; i++) {
                     if (Double.compare(i * stepSize.doubleValue(), newValue.doubleValue()) <= 0) {
-                        ledList.get(i).setOn(true);
+                        if (newValue.doubleValue() == 0.0) {
+                            ledList.get(i).setOn(false);
+                        } else {
+                            ledList.get(i).setOn(true);
+                        }
```

Comment:
Seems like  this would have more efficient 
  ledList.get(i).setOn(newValue.doubleValue() != 0.0);

But that is just syntax. :-)
Gerrit no longer supports this code, so I'll merge it.